src/main/java/uk/gov/register/providers/ThrowableExceptionMapper.java

```diff
@@ -35,7 +35,15 @@ public Response toResponse(Throwable exception) {
             return Response.status(Response.Status.NOT_FOUND)
                     .header(HttpHeaders.CONTENT_TYPE, ExtraMediaType.TEXT_HTML)
                     .entity(viewFactory.thymeleafView("404.html"))
-                    .build();        }
+                    .build();
+        }
+
+        if (exception instanceof UnsupportedOperationException) {
```

Comment:
don't like this 👎  there is library code that can throw `UnsupportedOperationException`